ios/BetterSocial.xcodeproj/project.pbxproj

```diff
@@ -326,34 +323,33 @@
 			isa = PBXFrameworksBuildPhase;
 			buildActionMask = 2147483647;
 			files = (
-				2E7357E3EDC718B14778FD80 /* BuildFile in Frameworks */,
+				2E7357E3EDC718B14778FD80 /* (null) in Frameworks */,
 				895588D8DAA81DCBB9A9EDA0 /* libPods-BetterSocial-tvOSTests.a in Frameworks */,
 			);
 			runOnlyForDeploymentPostprocessing = 0;
 		};
-		888DAA1A29B48C8900311304 /* Frameworks */ = {
+		3F83D1AAB2844DCF34EE511C /* Frameworks */ = {
 			isa = PBXFrameworksBuildPhase;
 			buildActionMask = 2147483647;
 			files = (
-				18A175CE91DE33A7AC31D46F /* libPods-BetterSocial-BetterSocialStaging.a in Frameworks */,
+				036832A7C6B969325240D615 /* libPods-OneSignalNotificationServiceExtension.a in Frameworks */,
 			);
 			runOnlyForDeploymentPostprocessing = 0;
 		};
-		889CAB6427D46CD6007F5892 /* Frameworks */ = {
+		888DAA1A29B48C8900311304 /* Frameworks */ = {
 			isa = PBXFrameworksBuildPhase;
 			buildActionMask = 2147483647;
 			files = (
-				92D8142362E828E0BFD1A9D0 /* BuildFile in Frameworks */,
-				140B3EFDAF3C6F6A17D21477 /* libPods-BetterSocial-BetterSocialDev.a in Frameworks */,
+				18A175CE91DE33A7AC31D46F /* libPods-BetterSocial-BetterSocialStaging.a in Frameworks */,
 			);
 			runOnlyForDeploymentPostprocessing = 0;
 		};
-		C4AE08FF2AC43A4800E13758 /* Frameworks */ = {
+		889CAB6427D46CD6007F5892 /* Frameworks */ = {
 			isa = PBXFrameworksBuildPhase;
 			buildActionMask = 2147483647;
 			files = (
-				962349FAE6BEC6C5CCF1D8EB /* BuildFile in Frameworks */,
-				0E0E0A4F064AF84E69E81068 /* libPods-OneSignalNotificationServiceExtension.a in Frameworks */,
+				92D8142362E828E0BFD1A9D0 /* (null) in Frameworks */,
+				140B3EFDAF3C6F6A17D21477 /* libPods-BetterSocial-BetterSocialDev.a in Frameworks */,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

There are changes in the Frameworks section where `(null) in Frameworks` is added. This seems like an error or a placeholder that was accidentally committed. It should be replaced with the correct framework reference.

```diff
- 2E7357E3EDC718B14778FD80 /* BuildFile in Frameworks */,
+ 2E7357E3EDC718B14778FD80 /* (null) in Frameworks */,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->